framework/include/fwk_arch.h

```diff
@@ -222,6 +222,15 @@ struct fwk_arch_init_driver {
  */
 int fwk_arch_init(const struct fwk_arch_init_driver *driver);
 
+/*!
+ * \brief Stop the framework library.
+ *
+ * \retval ::FWK_SUCCESS Operation succeeded.
+ * \retval ::FWK_E_PARAM One or more parameters were invalid.
+ * \retval ::FWK_E_PANIC Unrecoverable initialization error.
```

Comment:
This is not doing the initialization. You can say `Unrecoverable` or `generic` error.

Reply:
ok

Reply:
yes, forgot to clean the copy/paste of fwk_arch_init